manim/mobject/types/point_cloud_mobject.py

```diff
@@ -21,7 +21,7 @@
 
 
 class PMobject(Mobject):
-    def __init__(self, stroke_width=DEFAULT_STROKE_WIDTH, **kwargs):
+    def __init__(self, stroke_width=DEFAULT_STROKE_WIDTH, radius=2.0, **kwargs):
```

Comment:
This argument shouldn't be added. Instead `PointCloudMobject` shouldn't pass its `radius` argument up to `Mobject1D`.